source/controlTypes.py

```diff
@@ -638,6 +641,20 @@
 }
 
 def processPositiveStates(role, states, reason, positiveStates):
+	"""Processes the states for an object and returns the positive states to output for a specified reason.
+	For example, if C{STATE_CHECKED} is among the returned states, it means that the processed object is checked.

```

Comment:
Trivial I guess, but "in the returned states" is a bit closer to what you'd actually write in a program to check this.